test/integration/scheduler/scheduler_test.go

```diff
@@ -1075,6 +1075,43 @@ var _ = ginkgo.Describe("Scheduler", func() {
 			util.FinishWorkloads(ctx, k8sClient, pWl3)
 			util.ExpectWorkloadsToBeAdmitted(ctx, k8sClient, devCQ.Name, dWl1)
 		})
+
+		ginkgo.It("Should schedule when fungibility is set", func() {
+			prodCQ = testing.MakeClusterQueue("prod-cq").
+				Cohort("all").
+				ResourceGroup(*testing.MakeFlavorQuotas("on-demand").Resource(corev1.ResourceCPU, "10", "10").Obj()).
+				Obj()
+			gomega.Expect(k8sClient.Create(ctx, prodCQ)).Should(gomega.Succeed())
+
+			devCQ = testing.MakeClusterQueue("dev-cq").
+				Cohort("all").
+				FlavorFungibility(kueue.FlavorFungibility{WhenCanBorrow: kueue.TryNextFlavor}).
```

Comment:
Add another test for the other preemption policy.

Reply:
sure. done